pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaMPI.hpp

```diff
@@ -1617,6 +1650,54 @@ class StateVectorCudaMPI final
         mpi_manager_.Barrier();
     }
 
+    /**
+     * @brief collapse the state vector to a given basis state.
+     *
+     * @param wire_local Local wire index.
+     * @param branch Branch index.
+     */
+    void collapse_local_(const int wire_local, const bool branch) {
+        cudaDataType_t data_type;
+
+        if constexpr (std::is_same_v<CFP_t, cuDoubleComplex> ||
+                      std::is_same_v<CFP_t, double2>) {
+            data_type = CUDA_C_64F;
+        } else {
+            data_type = CUDA_C_32F;
+        }
+
+        std::vector<int> basisBits(1, wire_local);
+
+        double abs2sum0_local, abs2sum1_local;
```

Comment:
Following the C++ style guideline:
```suggestion
        double abs2sum0_local;
        double abs2sum1_local;
```